packages/front-end/pages/features/index.tsx

```diff
@@ -199,84 +220,89 @@ export default function FeaturesPage() {
               </tr>
             </thead>
             <tbody>
-              {sorted.slice(start, end).map((feature) => {
-                let rules = [];
-                environments.forEach(
-                  (e) => (rules = rules.concat(getRules(feature, e.id)))
-                );
+              {sorted
+                .filter((f) => !f.archived || showArchived)
+                .slice(start, end)
+                .map((feature) => {
+                  let rules = [];
+                  environments.forEach(
+                    (e) => (rules = rules.concat(getRules(feature, e.id)))
+                  );
 
-                // When showing a summary of rules, prefer experiments to rollouts to force rules
-                const orderedRules = [
-                  ...rules.filter((r) => r.type === "experiment"),
-                  ...rules.filter((r) => r.type === "rollout"),
-                  ...rules.filter((r) => r.type === "force"),
-                ];
+                  // When showing a summary of rules, prefer experiments to rollouts to force rules
+                  const orderedRules = [
+                    ...rules.filter((r) => r.type === "experiment"),
+                    ...rules.filter((r) => r.type === "rollout"),
+                    ...rules.filter((r) => r.type === "force"),
+                  ];
 
-                const firstRule = orderedRules[0];
-                const totalRules = rules.length || 0;
+                  const firstRule = orderedRules[0];
+                  const totalRules = rules.length || 0;
 
-                const isDraft = !!feature.draft?.active;
-                let version = feature.revision?.version || 1;
-                if (isDraft) version++;
+                  const isDraft = !!feature.draft?.active;
+                  let version = feature.revision?.version || 1;
+                  if (isDraft) version++;
 
-                return (
-                  <tr key={feature.id}>
-                    <td>
-                      <Link href={`/features/${feature.id}`}>
-                        <a>{feature.id}</a>
-                      </Link>
-                    </td>
-                    <td>
-                      <SortedTags tags={feature?.tags || []} />
-                    </td>
-                    {toggleEnvs.map((en) => (
-                      <td key={en.id} className="position-relative">
-                        <EnvironmentToggle
-                          feature={feature}
-                          environment={en.id}
-                          mutate={mutate}
-                        />
+                  return (
+                    <tr key={feature.id}>
+                      <td>
+                        <Link href={`/features/${feature.id}`}>
+                          <a style={feature.archived ? { color: "red" } : null}>
```

Comment:
That makes sense why you didn't do the badge.  My issue with using red text is that red usually implies an error.

Looking at metrics, we're doing two things for archived rows:
1.  Lower opacity for the whole row: `<tr className={metric.status === "archived" ? "text-muted" : ""}`
2. Adding an additional "status" column that shows either "active" or "archived" when the "show archived" toggle is on

The features table is a lot more crowded than metrics already, so maybe just do the first change (`text-muted` class on the row) instead of changing the text color.